pyro/contrib/oed/eig.py

```diff
@@ -334,7 +335,22 @@ def loss_fn(design, num_particles):
     return loss_fn
 
 
-class EwmaLog(torch.autograd.Function):
+class _EwmaLogFn(torch.autograd.Function):
```

Comment:
Can you add a dosctring explaining how this class differs from the usual `torch.log()`?

Reply:
We have doc string at EwmaLog though. I didn't change it so it might not show up in git diff.